### PR TITLE
feat(avatar-carousel): avatar carousel & navigation & localization

### PR DESCRIPTION
**Pull request by VladislavRiabko**
- add OnlyImage component 
- add new preset on Carousel component 
- rename subTitleID to subtitleKey
- rename titleID to titleKey
- refactor ParentsOnboardingStackNavigator 
- fix disable logic button on LanguageSelectionScreen
- fix Android swipe 
- add ReadonlyArray 
- refactor LanguageSelection screen 
- refactor RadioButtonList component

### CODE REVIEW

File: app/screens/Onboarding/Parents/LanguageSelection/LanguageSelection.tsx [hunk 1]
```diff
@@ -22,22 +21,24 @@ export const LanguageSelectionScreen: React.FC<ILanguageSelectionScreenProps> =
   ({ navigation }) => {
     const disptach = useAppDispatch();
 
-    const [selectedLanguage, setSelectedLanguage] = useState<string[]>(['en']);
+    const [selectedLanguage, setSelectedLanguage] = useState<string[]>(['']);
 
     const { t } = useTranslation();
 
-    useEffect(() => {
+    const onConfirm = useCallback(() => {
```
Comment (Reykjavik151) [inline app/screens/Onboarding/Parents/LanguageSelection/LanguageSelection.tsx]: rename to `onSubmit`

File: app/screens/Onboarding/Parents/OnboardingCarousel/OnboardingCarousel.data.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { IMAGES } from '../../../../assets';
+import { ICarouselItem } from '../../../../components';
+
+// TODO: change image for correct
+export const CHARMS_CAROUSEL: ICarouselItem[] = [
```
Comment (Reykjavik151) [inline app/screens/Onboarding/Parents/OnboardingCarousel/OnboardingCarousel.data.ts]: ReadonlyArray?

File: app/screens/Onboarding/Parents/OnboardingCarousel/OnboardingCarousel.tsx [hunk 3]
```diff
@@ -13,13 +12,18 @@ import { CHARMS_CAROUSEL } from './OnboardingCarousel.data';
 import { IOnboardingCarouselScreenProps } from './OnboardingCarousel.props';
 
 export const OnboardingCarouselScreen: React.FC<IOnboardingCarouselScreenProps> =
-  () => {
+  ({ navigation }) => {
     const { t } = useTranslation();
 
+    const onNextPress = useCallback(() => {
```
Comment (Reykjavik151) [inline app/screens/Onboarding/Parents/OnboardingCarousel/OnboardingCarousel.tsx]: rename to `onSubmit`

File: app/screens/Onboarding/Parents/SignUp/SignUpAvatar/SignUpAvatar.data.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import { IMAGES } from '../../../../../assets';
+import { ICarouselItem } from '../../../../../components';
+
+// TODO: change image for correct
+export const SING_UP_CAROUSEL_DATA: ICarouselItem[] = [
```
Comment (Reykjavik151) [inline app/screens/Onboarding/Parents/SignUp/SignUpAvatar/SignUpAvatar.data.ts]: ReadonlyArray

**Review by Reykjavik151 — CHANGES_REQUESTED** [review]

File: app/screens/Onboarding/Parents/SignUp/SignUpAvatar/SignUpAvatar.data.ts (hunk 4 above)
Comment (emberglaz) [reply app/screens/Onboarding/Parents/SignUp/SignUpAvatar/SignUpAvatar.data.ts]: `SING_UP_CAROUSEL`

File: app/screens/Onboarding/Parents/OnboardingCarousel/OnboardingCarousel.data.ts (hunk 2 above)
Comment (emberglaz) [reply app/screens/Onboarding/Parents/OnboardingCarousel/OnboardingCarousel.data.ts]: in another pr

File: app/screens/Onboarding/Parents/LanguageSelection/LanguageSelection.data.ts [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import { IRadioButtonListItem } from '../../../../components';
 
-export const LANGUAGES: ReadonlyArray<IRadioButtonListItem> = [
+export const LANGUAGES: IRadioButtonListItem[] = [
```
Comment (Reykjavik151) [inline app/screens/Onboarding/Parents/LanguageSelection/LanguageSelection.data.ts]: ReadonlyArray is needed, so leave only `id` and `title`
The `isActive` you need map later with these Readonly values

File: app/screens/Onboarding/Parents/OnboardingCarousel/OnboardingCarousel.data.ts (hunk 2 above)
Comment (Reykjavik151) [inline app/screens/Onboarding/Parents/OnboardingCarousel/OnboardingCarousel.data.ts]: ReadonlyArray?

**Review by Reykjavik151 — CHANGES_REQUESTED** [review]

**Comment by Reykjavik151** [comment]
Also, please fix your typing mistake in the PR naming -- `carousel`

**Review by Reykjavik151 — APPROVED** [review]

**Pull request merged**